code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -40,8 +40,7 @@
 			filter_types += gas_id2path(f)
 
 /obj/machinery/atmospherics/components/unary/vent_scrubber/on
-	on = TRUE
-	icon_state = "scrub_map_on"
+	desc = "This scrubbers seem particularly eager to help!"
```

Comment:
I think this is incorrect plural

Reply:
Yea I saw, im missing quite a bit of sleep, its just a placeholder for testing.